apps/condo/domains/resident/access/Resident.js

```diff
@@ -18,8 +19,24 @@ async function canReadResidents ({ authentication: { item: user } }) {
 async function canManageResidents ({ authentication: { item: user }, originalInput, operation, itemId, context }) {
     if (!user) return false
     if (user.isAdmin) return true
-    if (operation === 'create' && user.type === RESIDENT) {
-        return true
+    if (user.type === RESIDENT) {
+        if (operation === 'create') return true
```

Comment:
We have register resident ? We don’t need to create it

Reply:
Hmm, I thought, that access control hooks are being called, when executing custom mutation (with passing not modified context). I see, that they are bypassed.

Reply:
Removed in `dc1fc488`.

Reply:
Fixed in `058686b1`